### ISSUE TITLE
13. Feedback Implementation

### ISSUE THREAD

**Issue by CYCBrian** [CYCBrian/Week-12-Employee-Tracker-CMS#28]
Suggestions for improvement for add role, employee, and update employee's manager, instead of using the id, display the actual department, manager name.

**Comment by CYCBrian** [comment]
It also prevents the user from mistakenly inputting a invalid id and having to redo the process again.

**Comment by CYCBrian** [comment]
LOTS OF CHANGES:

- previously, my get roles, employees, and managers functions would only return the ids of the department, roles, and managers. I've now added join statements to get it to display the actual name and titles instead of just the ids for easier viewing. This was the hardest, most confusing part of the update.

- for adding a role, i've changed input to list for adding a department so that there is no chance the user the can input a non existent id and be given an error.

- for adding an employee, instead of having the user input the role id, I've changed it to a list that uses the getRoles function. The same is done for selecting the employee's manager. For managers. I've also pushed an extra option of "None" so user aren't forced to add a manager if the employee they are adding is suppose to be a manager anyways. This is accomplished by wrapping the mapping of the department key value pairs in a const then pushing a new key value pair of name: "None", value: null into the const, then returning it for use in the prompt.

- for updating an employee, I've also changed the code a bit and pushed an option to select "None" if the employee has become an manager.

- for all the delete functions. I've change the code a bit as well and also pushed an option to select "None" if the user decided to not remove any department, role, or employee.
